### DISCUSSION TITLE
Daily 04/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#3] [Daily]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué hice o haré hoy?
Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve ordenando la wiki, la documentación y aguegando algunas historias de usuarios. El sábado por la mañana participe de la weekly.
Hoy cree las issue para las historias de usuarios y la tarea respectiva da casa una, habrá que revisar si de cada tarea se desprenden subtareas. Seguiré revisando la documentación para ver que mas se puede agregar.

**Comment by BraianTroncoso** [comment]
Estoy estudiando android para adelantar un poco. Me puse a ver patrones de diseños para seleccionar uno, estoy entre mvc y mvvc. También hoy tuve clases de Python.

**Comment by Valentino399** [comment]
Este fin de semana estuve averiguando el uso de android studio y viendo videos en youtube con respecto a ese programa

**Comment by marceloedelgado** [comment]
El fin de semana estuve estudiando Android Studio y Java haciendo pruebas de vistas de menu y comportamiento de botones
